### DISCUSSION TITLE
Tell me your favorite technical or specialized book

### DISCUSSION THREAD

**Discussion by kohanaist** [NAIST-SE/AttractingContributors#1027] [General]
https://github.com/kohanaist/AttractingContributors-technical-or-specialized-book

Please share with me some interesting technical books or specialized books.

**Comment by oyewaleoyebode** [comment]
Game of thrones

**Comment by Canppchk** [comment]
I add [Monolith to Microservices , O'Reilly]  on your list.
could you help me with my project too? Thanks!!
https://github.com/Canppchk/AttractingContributors-Ramen

**Comment by mustafizur-r** [comment]
Hello [@kohanaist ](https://github.com/kohanaist)  , I recently made a contribution to your project , I have added "The Alchemist by Paulo Coelho " could you please contribute to my as well ? Following is the link to [my repository](https://github.com/MustafizurRU/AttractingContributors-My_Favorite_Sci-Fi_Books) . Thanks in advance.

**Comment by anamhamza786** [comment]
Hey,
I sent you a pull request. can you please send me a pull request.

Please Add your Favourite Players name here.
Here is the link to my forked repository: https://github.com/anamhamza786/AttractingContributors-cricketplayers.git
